### DISCUSSION TITLE
How to run without checkpoint

### DISCUSSION THREAD

**Discussion by BinaryQuantumSoul** [vladmandic/sdnext#826] [Q&A]
Hello,
I'd like to use the [Stable Horde extension](https://github.com/natanjunges/stable-diffusion-webui-stable-horde) in order to remotely generate images. 

They say to use `--ui-debug-mode` in order to prevent checkpoint loading, but since the command line arguments changed in this repo, I don't know how to do it.

Thanks

**Comment by vladmandic** [comment]
the ui-debug mode was buggy, i've removed it.
i'll add option later today to run without checkpoint. in order to do that, use `--ckpt none`.
new code will be online in ~4h.

**Reply by BinaryQuantumSoul** [reply]
Thank you
